lib/cocktail/validation/time_of_day.ex

```diff
@@ -17,7 +17,7 @@ defmodule Cocktail.Validation.TimeOfDay do
   @spec next_time(t, Cocktail.time(), Cocktail.time()) :: Cocktail.Validation.Shift.result()
   def next_time(%__MODULE__{times: times}, time, _) do
```

Comment:
@wakarana1 this means that `[]` was passed in for `times`, it might make more sense to have a function clause for this instead of just falling back to `nil` inside the function and letting the rest of the function figure it out